app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -1,11 +1,46 @@
-import React from "react"
+import React from "react";
+import {
+  Card,
+  CardBody,
+  CardTitle,
+  CardText,
+  ListGroup,
+  ListGroupItem,
+  CardLink,
+  Button,
+} from "reactstrap";
 
-const ApartmentIndex = () => {
+const ApartmentIndex = ({ mockApartments }) => {
   return (
-    <>
-      <h3>ApartmentIndex</h3>
-    </>
-  )
-}
+    <div>
+      {mockApartments?.map((apartments, index) => {
```

Comment:
I would recommend changing your naming convention from mockApartments to just apartments since you are passing apartments from state and not necessarily mockApartments. Not a huge deal at this point, but may cause confusion down the road.

Reply:
great point.